ray-operator/config/samples/ray-service.tpu-multi-host.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: ray.io/v1
+kind: RayService
+metadata:
+  name: s-d-tpu
+spec:
+  serveConfigV2: |
+    applications:
+      - name: stable_diffusion
+        import_path: stable_diffusion.stable_diffusion_tpu:deployment
+        runtime_env:
+          working_dir: "https://github.com/ryanaoleary/serve_config_examples/archive/refs/heads/tpu-example.zip"
```

Comment:
Leaving a note here so we don't forget to revert the fork back to kuberay instead of ryanaoleary 

I'm assuming this was intentional for testing purposes

Reply:
Done in [98c5312](https://github.com/ray-project/kuberay/pull/2198/commits/98c531208f3e4d2b2bc6901862a026d10376568b).